components/Workspaces/TrackedRepoWizard/SearchByReposStep.tsx

```diff
@@ -0,0 +1,85 @@
+import { FaSearch } from "react-icons/fa";
+import Search from "components/atoms/Search/search";
+import { Avatar } from "components/atoms/Avatar/avatar-hover-card";
+import { SearchedReposTable } from "../SearchReposTable";
+import { TrackedRepoWizardLayout } from "./TrackedRepoWizardLayout";
+
+interface SearchByReposStepProps {
+  onAddToTrackingList: () => void;
+  onCancel: () => void;
+  onSearch: (search?: string) => void;
+  trackedReposCount: number;
+  repositories: {
+    owner: string;
+    name: string;
+  }[];
+  suggestedRepos: {
+    owner: string;
+    name: string;
+  }[];
+}
+
+const EmptyState = () => {
+  return (
+    <div className="grid place-content-center">
+      <div className="text-center flex flex-col items-center p-24 gap-2 max-w-lg">
+        <div className="p-3 border rounded-lg mb-2">
+          <FaSearch size={24} className="text-light-slate-9" />
+        </div>
+        <span className="font-semibold">No repositories added yet!</span>
+        <span>Use the search bar to find the repositories you want to track on your workspace.</span>
+      </div>
+    </div>
+  );
+};
+
+export const SearchByReposStep = ({
+  onAddToTrackingList,
+  onCancel,
+  onSearch,
+  trackedReposCount,
+  repositories,
+  suggestedRepos,
+}: SearchByReposStepProps) => {
+  // TODO: Implement these functions
+  const onFilterRepos = () => {};
+  const onSelectRepo = () => {};
```

Comment:
These will get implemented once this gets wired up to the tracked repos flow of workspace creation and will most likely be props passed in. For now these components only live in Storybook, so not a big deal.